ark/utils/segmentation_utils.py

```diff
@@ -127,3 +129,55 @@ def concatenate_csv(base_dir, csv_files, column_name="point", column_values=None
             combined_data = pd.concat((combined_data, temp_data), axis=0, ignore_index=True)
 
     combined_data.to_csv(os.path.join(base_dir, "combined_data.csv"), index=False)
+
+
+def visualize_watershed_transform(segmentation_labels_xr, channel_data_xr,
+                                  output_dir,
+                                  overlay_channels, fovs=None,
+                                  save_tifs='overlays'):
+    """Runs the watershed transform over a set of probability masks output by deepcell network
+    Saves xarray to output directory
+    Args:
```

Comment:
Not sure if ReadTheDocs complained about this or rendered it badly, but you should have an extra blank line in between the description and the Args list. I'll look into including a check for these in docs/conf.py.